lib/kbsecret/cli/kbsecret-cp

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require "kbsecret"
+require "tty-prompt"
+
+include KBSecret
+
+cmd = CLI.create do |c|
+  c.slop do |o|
+    o.banner = <<~HELP
+      Usage:
+        kbsecret cp [options] <target-session> <record> [record]
+    HELP
+
+    o.string "-s", "--session", "the session containing the record", default: :default
+    o.bool "-m", "--move", "delete the record after copying"
+  end
+
+  c.dreck do
+    string :target_session
+    string :label
+  end
+
+  c.ensure_session!
+end
+
+target_session_label    = cmd.args[:target_session]
```

Comment:
It'd be okay to drop this variable and just use `cmd.args[:target]` directly below.

Reply:
👍